### ISSUE TITLE
Rename 'master' branch to 'main'

### ISSUE THREAD

**Issue by aiuto** [bazelbuild/bazel-website#274]

**Comment by aiuto** [comment]
@floriographygoth @fweikert 
Is there anything blocking us from just doing this?

**Comment by fweikert** [comment]
Not really, we just need to make sure to update the settings of https://buildkite.com/bazel-trusted/docgen-bazel-website

**Comment by floriographygoth** [comment]
Sounds good to me. I'll send an email of how this may impact the docs, but I'm on board with this change.

**Comment by aiuto** [comment]
@fweikert   Can you take this over?  You're the one who knows the pipelines.
